src/PushNotifications.Contracts/PushNotifications/Delivery/IPushNotificationDelivery.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace PushNotifications.Contracts.PushNotifications.Delivery
+{
+    public interface IPushNotificationDelivery
```

Comment:
maybe combine this interface with IPushNotificationBufferedDelivery

Reply:
Thx, no thx